frontend/packages/kubevirt-plugin/src/components/create-vm-wizard/redux/stateUpdate/vmSettings/vm-settings-tab-state-update.ts

```diff
@@ -45,20 +44,11 @@ export const selectUserTemplateOnLoadedUpdater = ({
     return;
   }
 
-  const iUserTemplates = iGetLoadedData(iUserTemplatesWrapper);
-
-  const firstUserTemplateName = ignoreCaseSort(
-    iUserTemplates
-      .toIndexedSeq()
-      .toArray()
-      .map(iGetName),
-  )[0];
-
   dispatch(
     vmWizardInternalActions[InternalActionType.UpdateVmSettings](id, {
       [VMSettingsField.USER_TEMPLATE]: {
         initialized: true,
-        value: firstUserTemplateName,
+        value: '',
```

Comment:
Can be `undefined`, but the empty string conforms farther flows.

Reply:
is there a reason to not delete whole `selectUserTemplateOnLoadedUpdater` ?

Reply:
OK, i see, we still want to set initialized when templates are loaded but we should rename the function to better reflect what its doing now (there's no selection happening anymore)